inc/com/centreon/engine/notification.hh

```diff
@@ -59,6 +59,8 @@ class notification {
   notifier::reason_type get_reason() const;
   uint32_t get_notification_interval() const;
   bool sent_to(const std::string& user) const;
+  void insert_contacts(std::set<std::string> contact_notified);
```

Comment:
I would prefer add_contacts() which is more explicit on the fact we add contacts and not juste replace them.